global-script.lua

```diff
@@ -3874,7 +3874,15 @@ end
 -- Play spirit
 function onClickedPlaySpirit(target_obj, source_color, alt_click)
     local target_color = target_obj.getVar("playerColor")
-    swapPlayerColors(source_color, target_color)
+    if swapPlayerColors(source_color, target_color) then
+        Wait.frames(function ()
+            Player[target_color].lookAt({
+                position = playerBlocks[target_color].getPosition() - Vector(0,0,15),
+                pitch = 70,
+                distance = 30,
+            })
+        end, 2)
```

Comment:
why wait 2 frames instead of 1?

Reply:
It appears that switching the camera when switching colors happens after this, with a 1 frame delay. I'm not sure what happens for a remote player, when playing multiplayer; it may be even longer there, but I don't have a way to test.